src/popup/pages/CurrencySettings.vue

```diff
@@ -1,34 +1,36 @@
 <template>
   <div class="currency-settings">
-    <div class="description">
+    <p class="text-description">
       {{ $t('pages.currencySettings.chooseLanguage') }}
-    </div>
+    </p>
 
-    <RadioButton
-      v-for="{ code, name, symbol }, index in currencies"
-      :key="`${index}-${code}`"
-      :value="current && current.currency == code"
-      :disabled="false"
-      :class="['currency', {active: current && current.currency == code}]"
-      @input="switchCurrency(code)"
-    >
-      <div
-        class="row"
-        @click="switchCurrency(code)"
+    <div class="options">
+      <RadioButton
+        v-for="{ code, name, symbol }, index in currencies"
+        :key="`${index}-${code}`"
+        :value="current && current.currency == code"
+        :disabled="false"
```

Comment:
:disabled="false" normaly is always false

Reply:
Done